main/config.go

```diff
@@ -95,11 +97,22 @@ func (c *Config) Load(configDir string, filename string) error {
 		log.SetFormatter(&log.TextFormatter{FullTimestamp: true, TimestampFormat: "2006-01-02 15:04:05.000 -0700"})
 	}
 
-	_ = c.loadAuthMap(configDir)
+	err = c.loadDeviceIdentitiesFile(configDir)
+	if err != nil {
+		return err
+	}
+
+	err = c.loadAuthMap(configDir) // legacy
+	if err != nil {
+		return err
+	}
```

Comment:
I understand here the Auth map takes precedence before the Identities File.

would it make sense to automatically create an identity file from an Auth map, in case it was found?
If the authMap is deprecated, This would make the migration to the auth map a little smoother.

Reply:
the Auth map probably should not take precedence before the Identities File in case they are both present. i will switch the order.

Reply:
I am not sure if I understand your second remark correctly, but note that the auth map contains private keys and I dont think we should write them into a file. please, lets discuss this further in person.

Reply:
OK, that is a good point. fine with me